pkgs/development/python-modules/radicale_infcloud/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "radicale_infcloud";
+  name = "${pname}-${version}";
+  version = "972757bf4c6be8b966ee063e3741ced29ba8169f";
+
+  src = fetchFromGitHub {
+    owner = "Unrud";
+    repo = "RadicaleInfCloud";
+    rev = version;
+    sha256 = "1c9ql9nv8kwi791akwzd19dcqzd916i7yxzbnrismzw4f5bhgzsk";
+  };
+
+  doCheck = false; # Tries to import radicale, circular dependency
```

Comment:
But radicale does not depend on this, right?
How does the circular dependency happen?

Reply:
This is a plugin for radicale, so I haven't enabled it for default but `radicale_infcloud` would normally be added as a dependency for `radicale`, which would cause the circular dependency.

Also there don't seem to actually be any tests in the repository, so I don't think this is really missing anything.